src/components/doi/UpdateRequestDialog.js

```diff
@@ -0,0 +1,237 @@
+                                    style={{
+                                        position: "absolute",
+                                        top: "50%",
+                                        left: "50%",
+                                    }}
+                                />
+                            )}
+
+                            {!isRequestFetching && (
+                                <>
+                                    <Field
+                                        name="email"
+                                        label={t("email")}
+                                        required={false}
+                                        margin="dense"
+                                        id={build(baseId, ids.EMAIL)}
+                                        component={FormTextField}
+                                    />
+
+                                    <Field
+                                        name="status"
+                                        validate={(value) =>
+                                            nonEmptyField(value, i18nUtil)
+                                        }
+                                    >
+                                        {({
+                                            field: { onChange, ...field },
+                                            ...props
+                                        }) => (
+                                            <FormSelectField
+                                                {...props}
+                                                label={t("status")}
+                                                required
+                                                field={field}
+                                                onChange={(event) => {
+                                                    onChange(event);
+                                                }}
+                                                id={build(baseId, ids.STATUS)}
+                                                size="small"
+                                            >
+                                                {Object.values(
+                                                    RequestStatus
+                                                ).map((status, index) => (
+                                                    <MenuItem
+                                                        key={index}
+                                                        value={status}
+                                                        id={baseId}
+                                                    >
+                                                        {status}
+                                                    </MenuItem>
+                                                ))}
+                                            </FormSelectField>
+                                        )}
+                                    </Field>
```

Comment:
I'm not sure I understand why this `Field` is using the `FormSelectField` this way.

The `FormSelectField` was implemented so that it can be used like this:
https://github.com/cyverse-de/sonora/blob/60d5cb0370cbd33b045df40d3fc9e0ed7aecea76/src/components/apps/launch/ResourceRequirements.js#L113-L124
or like this:
https://github.com/cyverse-de/sonora/blob/60d5cb0370cbd33b045df40d3fc9e0ed7aecea76/src/components/tools/edit/ToolRestrictions.js#L123-L143

So there shouldn't be a need to manually call `onChange` in the select field. For example:
```suggestion
                                    <Field
                                        id={build(baseId, ids.STATUS)}
                                        name="status"
                                        label={t("status")}
                                        required
                                        validate={(value) =>
                                            nonEmptyField(value, i18nUtil)
                                        }
                                        size="small"
                                        component={FormSelectField}
                                    >
                                                {Object.values(
                                                    RequestStatus
                                                ).map((status, index) => (
                                                    <MenuItem
                                                        key={index}
                                                        value={status}
                                                        id={baseId}
                                                    >
                                                        {status}
                                                    </MenuItem>
                                                ))}
                                    </Field>
```

Alternatively, we can also use an outlined `FormTextField` with a `select` prop instead (I actually think its UX is a little nicer then the `Select` field):
```suggestion
                                    <Field
                                        id={build(baseId, ids.STATUS)}
                                        name="status"
                                        label={t("status")}
                                        required
                                        validate={(value) =>
                                            nonEmptyField(value, i18nUtil)
                                        }
                                        component={FormTextField}
                                        select={true}
                                        variant="outlined"
                                        size="small"
                                    >
                                                {Object.values(
                                                    RequestStatus
                                                ).map((status, index) => (
                                                    <MenuItem
                                                        key={index}
                                                        value={status}
                                                        id={baseId}
                                                    >
                                                        {status}
                                                    </MenuItem>
                                                ))}
                                    </Field>
```

By the way, the `type` field in the `EditTool` form can be refactored in a similar way.

Reply:
Yup. I took it from https://github.com/cyverse-de/ui/blob/06c9fa859c8a366de30a8afa201231bee333eb50/react-components/src/tools/edit/EditTool.js#L178. But Agree `FormTextField` with `select` variant is nice. Will update it.